package.json

```diff
@@ -16,17 +16,14 @@
       "jquery": true
     },
     "globals": {
-      // 1. Table names
```

Comment:
Comments in package.json were causing an error while running the tests. Or maybe you mean removing the table names themselves from the announced global vars? In that case, it's because there's no reason for that if they are explicitly exported/imported which had to become the case to be able to use them in tests.